rfcs/text/0008-email.md

```diff
@@ -112,8 +115,8 @@ People
 
 The following are the people that consulted on the contents of this RFC.
 
-Jamie Hynds | author
-TBD | Sponsor
+Marius Iversen | Author
```

Comment:
We've been using GitHub handles in the `People` section as a convention for RFCs. Here's an [example](https://github.com/elastic/ecs/blob/master/rfcs/text/0001-wildcard-data-type.md#people).

Can we do the same here as well?